prime-router/src/test/resources/test_data_files/Hl7_ORU-R01/EHT-20210316-0001.hl7

```diff
@@ -1 +1 @@
SFT|Lifepoint^L^^^^&2.16.840.1.113883.9.265&ISO^XX^^^Lifepoint|1.2.1|Lifepoint|Lifepoint Binary ID||20200601|
PID|1||LT-306^^^ELLUME LIMITED&2.16.840.1.113883.9.265&ISO^PT^|||||||UNK^UNKNOWN^HL70005|^^SANDY^UT^84093^^^^|||||||||||UNK^UNKNOWN^HL70189||||||||N|
ORC|RE||1e837a04-7d87-4498-ac86-1476354ed257^ELLUME Limited^2.16.840.1.113883.9.265^ISO||CM|||||||ELLUME^^^^^^^^NPI&2.16.840.1.113883.4.6&ISO^^^^NPI||^^^^^^||||||||^^^^|^^^^^^|^^^^||||||||
OBR|1||1e837a04-7d87-4498-ac86-1476354ed257^ELLUME Limited^2.16.840.1.113883.9.265^ISO|94558-4^LN^94558-4^SARS-CoV-2 (COVID-19) Ag [Presence]in Respiratory specimen by Rapid immunoassay^L|||202103160000|||||||||ELLUME^^^^^^^^NPI&2.16.840.1.113883.4.6&ISO^^^^NPI|^^^^^^|||||20210316000000-0500|||F||||||||||
OBX|1|CWE|94558-4^LN^260415000^260415000^L|1|260415000^Not detected^SCT|||N^Normal^HL70078|||F|||202103160000|||Ellume COVID-19 Home Test|151002609|202103160000||||ELLUME Limited|57 Didsbury St^^East Brisbane^QLD^4169|^|
NTE|1||Presumptive Negative Test for SARS-CoV-2 (no antigen detected)|
SPM|1|^1e837a04-7d87-4498-ac86-1476354ed257&ELLUME Limited&2.16.840.1.113883.9.265&ISO||^871810001^
SCT||||^871810001^SCT|||||||||202103160000|202103160000|||||202103160000
\ No newline at end of file
+MSH|^~\&#|LIFEPOINT^2.16.840.1.114222.4.5.1^ISO|ELLUME Limited^2.16.840.1.113883.9.265^ISO|WATERS_^2.16.840.1.113883.9.265^ISO|WATERS_^2.16.840.1.113883.9.265^ISO|20210316162100-0500||ORU^R01^ORU_R01|658195889000001-1e837a04-7d87-4498-ac86-1476354ed257|P|2.5.1||||NE|USA||||PHLabReport-NoAck^HL7^2.16.840.1.113883.9.11^ISO|||||
SFT|Lifepoint^L^^^^&2.16.840.1.113883.9.265&ISO^XX^^^Lifepoint|1.2.1|Lifepoint|Lifepoint Binary ID||20200601|
PID|1||LT-306^^^ELLUME LIMITED&2.16.840.1.113883.9.265&ISO^PT^||pablo|||||UNK^UNKNOWN^HL70005|^^SANDY^UT^84093^^^^|||||||||||UNK^UNKNOWN^HL70189||||||||N|
ORC|RE||1e837a04-7d87-4498-ac86-1476354ed257^ELLUME Limited^2.16.840.1.113883.9.265^ISO||CM|||||||ELLUME^^^^^^^^NPI&2.16.840.1.113883.4.6&ISO^^^^NPI||^^^^^^||||||||test&test2&10^empty^NY^CA^|^^^^^^|^^^^||||||||
OBR|1||1e837a04-7d87-4498-ac86-1476354ed257^ELLUME Limited^2.16.840.1.113883.9.265^ISO|94558-4^LN^94558-4^SARS-CoV-2 (COVID-19) Ag [Presence]in Respiratory specimen by Rapid immunoassay^L|||202103160000|||||||||ELLUME^^^^^^^^NPI&2.16.840.1.113883.4.6&ISO^^^^NPI|^^^^^^|||||20210316000000-0500|||F||||||||||
OBX|1|CWE|94558-4^LN^260415000^260415000^L|1|260415000^Not detected^SCT|||N^Normal^HL70078|||F|||202103160000|||Ellume COVID-19 Home Test|151002609|202103160000||||ELLUME Limited^^^^^^^^^test|57 Didsbury St^^East Brisbane^QLD^4169|^|
NTE|1||Presumptive Negative Test for SARS-CoV-2 (no antigen detected)|
SPM|1|^1e837a04-7d87-4498-ac86-1476354ed257&ELLUME Limited&2.16.840.1.113883.9.265&ISO||^871810001^
SCT||||^871810001^SCT|||||||||202103160000|202103160000|||||202103160000

```

Comment:
I looked at the ObservationMessageTest output and I do not see warnings about the new data you added to both test files.  It is like it is not being extracted.  There should be new data in the CSV file for columns #40 and so on.

Reply:
I didn't realize that the ObservationMessageTest does a comparison between the hl7 and csv pair. Not sure why it hasn't failed... maybe a bug in there? Should I take a look to that ?

Reply:
Yes, take a look.  Do note I did move all that code and samples files to another location per an open PR, so some changes may be merged soon.